demo/data.js

```diff
@@ -1039,6 +1030,26 @@ export const demoData = {
         N8: { content: "FALSE" },
       },
     },
+    {
+      name: "Borders",
+      id: "sh4",
```

Comment:
don't use a number for the id.
If we later introduce other sheets before it won't make any sense and we will need to adapt it. Better if it can be avoided :)